example/data/input/account-schema-definition.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
let's put these XML files in a `datamapping` subfolder to match the Java code.

Reply:
done